schema.ts

```diff
@@ -87,6 +87,19 @@ export enum ValueHintType {
    * Indicates to interpret and render the value as a currency value.
    */
   Currency = 'currency',
+  /**
+   * Indicates to interpret and render the value as a file. The provided value should be a URL that
+   * points to a file. Coda will hotlink to the file when rendering it a doc.
+   *
+   * Using {@link FileAttachment} is recommended instead, so that the file is always accessible
+   * and won't appear as broken if the source file is later deleted.
+  */
+  FileReference = 'file',
```

Comment:
hmm im actually not sure if we need to support a reference for files. For images, this makes sense because you can directly embed an image but idk how that would work for files generally cc @jonathan-codaio

Reply:
Yeah, we don't support file references, that's unique to images (and maybe will go away eventually too).